include/functions_entries.inc.php

```diff
@@ -481,6 +481,7 @@ function &serendipity_fetchEntries($range = null, $full = true, $limit = '', $fe
     $ret =& serendipity_db_query($query, $fetch_single, 'assoc');
 
     if (is_string($ret)) {
```

Comment:
Probably accidentall, else it would also be information disclosure, we should only show that in debug envs

Reply:
Ah, the print. Right!